styles/designs/corn/parts/_exercises-shapes.scss

```diff
@@ -21,33 +18,10 @@
                                 map-merge($Exercise__Problem__Container,(
                                     _components:(
                                         $Exercise__ProblemSolution__Paragraph,
-                                        map-merge($Table--Eoc__Container, (
-                                            _components:(
-                                                $Table
-                                            )
-                                        )),
-                                    )
-                                )),
-                                map-merge($Exercise__Problem__Container--WithImageTableOnTop,(
-                                    _components:(
-                                        $Exercise__ProblemSolution__Paragraph,
-                                        map-merge($Table--Eoc__Container, (
-                                            _components:(
-                                                $Table
-                                            )
-                                        )),
                                     )
                                 )),
-                                map-merge($Exercise__Problem__Container--WithInlineListOnTop, (
-                                    _components: (
-                                        $Exercise__ProblemSolution__Paragraph,
-                                        map-merge($Table--Eoc__Container, (
-                                            _components:(
-                                                $Table
-                                            )
-                                        )),
-                                    )
-                                ))
+                                $Exercise__Problem__Container--WithImageTableOnTop,
+                                $Exercise__Problem__Container--WithInlineListOnTop
```

Comment:
`$Exercise__ProblemSolution__Paragraph` has been removed, from `$Exercise__Problem__Container--WithImageTableOnTop, $Exercise__Problem__Container--WithInlineListOnTop` because styling will be inherited from para inside regular `$Exercise__Problem__Container`.